Use JASP TextFields in Variables Window

boutinb opened the pull request:
Make the Variable Window more JASP Look & feel compatible,

boutinb committed:
Use JASP TextFields in Variables Window

boutinb committed:
Set focus on label input when double clicking

boutinb committed:
Highlight column and row header when cell is selected

JorisGoosen commented:
branchname wouldve been nice....

JorisGoosen commented:
<img width="1399" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/f617ee2b-763e-43e2-b09f-75f671c9a088">
Close button goes out of screen

JorisGoosen commented:
<img width="720" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/b5efc8c0-1956-4edd-ac2e-f7db4ec32337">
But ill fix it

JorisGoosen committed:
fix height
rounded textareas please if were gonna do jaspstyle everywhere
placeholdertext makes it slightly more inviting to type

JorisGoosen committed:
some more small improvements
